cmd/cluster/core/create.go

```diff
@@ -649,6 +649,9 @@ func (opts *RawCreateOptions) Validate(ctx context.Context) (*ValidatedCreateOpt
 			return nil, fmt.Errorf("could not retrieve kube clientset: %w", err)
 		}
 		if err := validateMgmtClusterAndNodePoolCPUArchitectures(ctx, opts, kc, &hyperutil.RegistryClientImageMetadataProvider{}); err != nil {
+			if strings.Contains(err.Error(), "failed to retrieve manifest") {
+				opts.Log.Info("WARNING: Unable to access the payload, skipping the Architectures check.", "error", err.Error())
```

Comment:
error says we are skipping, but we are still returning the error and `Validate()` fails right?

Reply:
Yeah, I must have misread something when I looked at this earlier. 

I think we need a `} else {` on L654